pkg/registry/registry_quay.go

```diff
@@ -0,0 +1,126 @@
+package registry
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	nurl "net/url"
+	"regexp"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/retagger/pkg/images"
+)
+
+// This code is largely lifted from nokia/docker-registry-client. It provides Quay-specific API behavior needed by retagger.
+
+var (
+	ErrNoMorePages = errors.New("No more pages")
```

Comment:
Same as above. This should be in an `errors.go` like elsewhere in our code. 

Same link: https://github.com/giantswarm/fmt/blob/master/go/errors.md#custom-errors

Reply:
Wrapped